src/Comments/Comments.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Comment from '../Comment/Comment'
+import '../scss/_Comments.scss'
+
+const Comments = ({ comments }) => {
+  const commentsList = comments.map(comment => {
+    return (
+      <Comment
+        key={comment.id}
+        comment={comment.comment}
+        author={comment.author}
+      />
+    )
+  })
+
+  return (
+    <section className='Comments-Container'>
+      {commentsList.reverse()}
```

Comment:
Love the reverse here! This is a super +1 for the user experience, so they can see their most recent comments at the top.